.github/workflows/main.yml

```diff
@@ -1,5 +1,11 @@
 name: Main
-on: push
+on:
+  push:
+    branches:
+      - main
```

Comment:
If this workflow only runs tests, we don't need to limit it to one branch. Also, the default branch is not `main` (yet).

Reply:
ok, changed to just [push, pull_request] so it runs regardless of branch for both events.